### PR TITLE
13 contact section 28.03.2023

### LINKED ISSUE #13: Contact section
**As a user,**
I want to see a "Contact" section after a "Discover why millions of entrepreneurs chose us" section.

### Links to design:
- [Figma web](https://www.figma.com/file/Pp48CPTdPVHY3Fkrxqn45b/Varbin-Tech_?node-id=466-410&t=F3nYB0RntM7HrE3F-4)
- [Figma mob](https://www.figma.com/file/Pp48CPTdPVHY3Fkrxqn45b/Varbin-Tech_?node-id=598-14006&t=F3nYB0RntM7HrE3F-4)

### Requirements:
- It should be a separate component
- We don't need a phone and address
- All icons in SVG format

### PR DESCRIPTION
**Pull request by InnaLazarenko**
### List of fixes:
- resolved #13

### CODE REVIEW

**Comment by var-bin** [comment]
@InnaLazarenko, we need to change `font-size` for the mobile view because of the issue

18-58-10-6yp31

File: components/contact/Contact.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+              </ContactLink>
+              <ContactLink
+                href={Settings.LinkedInLink}
+                underline="none"
+                target="_blank"
+                rel="noreferrer"
+                aria-label="Read more about Varbintech on LinkedIn"
+                sx={{ borderRadius: '2px' }}
+              >
+                <LinkedInIcon sx={{ fontSize: '26px' }} />
```
[the remark this answers is not in the document]
Comment (InnaLazarenko) [reply components/contact/Contact.tsx:88]: '+

File: components/contact/Contact.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                Flexible rendering and caching options, including Incremental Static Regeneration
+                (ISR), on a per-page level.
+              </ContactText>
+              <ContactIconContainer>
+                <RectangleBoxWhiteIcon />
+              </ContactIconContainer>
+            </div>
+
+            <Stack direction="row" spacing={3}>
+              <ContactRedditLink
```
[the remark this answers is not in the document]
Comment (InnaLazarenko) [reply components/contact/Contact.tsx:60]: '+

**Comment by var-bin** [comment]
@InnaLazarenko, it looks like we have some issues

11-13-16-ycmx8

**Review by var-bin — CHANGES_REQUESTED** [review]

File: components/contact/Contact.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  gitHubIcon: <GitHubIcon sx={{ fontSize: '28px' }} />,
+  linkedInIcon: <LinkedInIcon sx={{ fontSize: '32px' }} />,
+  twitterIcon: <TwitterIcon sx={{ fontSize: '26px' }} />,
+  upworkIcon: <UpworkSmallIcon />,
+  clutchIcon: <ClutchSmallIcon />,
+};
+
+const Contact: FC<ContactProps> = ({ data }) => {
+  return (
+    <PageContainer id="services">
```
[the remark this answers is not in the document]
Comment (InnaLazarenko) [reply components/contact/Contact.tsx]: '+

**Comment by InnaLazarenko** [comment]
> @InnaLazarenko, it looks like we have some issues
> 
> 11-13-16-ycmx8

+

**Review by var-bin — APPROVED** [review]

**Pull request merged**